cubequery/tasks/__init__.py

```diff
@@ -263,16 +263,14 @@ def standard_validation(self, args):
 
         errors = []
 
-        # Validates AOI
-        search = [p.name for p in self.parameters if p.d_type == DType.WKT]
-        for s in search:
-            errors = self.validate_standard_spatial_query(args[s])
-
         # Validates information against input_conditions.json
         for key in keys:
             for d in _settings_json[key]:
                 if d['name'] == args[key]:
                     for condition in d['conditions']:
+                        # Process specific conditions

```

Comment:
This needs more comments at the very least, I'm not entirely sure what this is for.